### DISCUSSION TITLE
Multi-window Support

### DISCUSSION THREAD

**Discussion by throwingdarts** [microsoft/azuredatastudio#16061] [Ideas]
In VS we can drag a tab to another screen allowing a quick way to see two files side-by-side. It would be very helpful to have this same functionality in data studio. Or even if we could open another data studio window and then drag a tab to the new window that would be helpful.

**Comment by Charles-Gagnon** [comment]
Could you give a specific example of what functionality you're looking for? VS Code doesn't seem to support dragging a file to a new screen and launching a new window of VS Code there as far as I can tell. And as far as dragging a tab to another instance of ADS goes that seems to work for me (at least for query windows). 

I'd also suggest that you open a [feature request](https://github.com/microsoft/azuredatastudio/issues/new?assignees=&labels=&template=feature_request.md&title=) if there's behavior you think should be added so that it can be triaged and tracked better.

**Reply by throwingdarts** [reply]
My post's comparison was VS, not VS Code. Also, when I drag a query tab from ADS to another screen it simply saves the sql file to the desktop. In VS it opens in a VS editor on the other screen. To do the same in ADS I have to open another instance of ADS and open the file separately in that instance which is time consuming. I can't even drag an open file from one instance of ADS to the other. I am not complaining about ADS (much better option for Azure than SSMS). Just seems that in this industry, where most use multiple monitors, it _could_ be a little more _mult-monitor friendly_.

@Charles-Gagnon - thanks for the link to create a feature request. https://github.com/microsoft/azuredatastudio/issues/16069#issue-942165378
